pkg/proxy/ipvs/proxier.go

```diff
@@ -670,25 +681,29 @@ func (handle *LinuxKernelHandler) GetKernelVersion() (string, error) {
 	return strings.TrimSpace(string(fileContent)), nil
 }
 
+// Store kernelVersion so we can run handler once and use the value to check if certain sysctls
+// should not be run on older kernels
+var kernelVersion *version.Version
+
 // CanUseIPVSProxier returns true if we can use the ipvs Proxier.
 // This is determined by checking if all the required kernel modules can be loaded. It may
 // return an error if it fails to get the kernel modules information without error, in which
 // case it will also return false.
-func CanUseIPVSProxier(handle KernelHandler, ipsetver IPSetVersioner) (bool, error) {
+func CanUseIPVSProxier(handle KernelHandler, ipsetver IPSetVersioner) (bool, string, error) {
```

Comment:
CanUseIPVSProxier returning the kernel version still seems weird. I think making KernelHandler as a parameter to NewProxier makes more sense.

Reply:
I agree that it is not ideal - are we ok with calling ```kernelVersionStr, err := handle.GetKernelVersion()``` twice?

Reply:
Yeah I think it's ok